contributors/sig-ownership.md

```diff
@@ -0,0 +1,51 @@
+# Sig Ownership Responsibilities
+
+## Overview
+
+This document describes the commitment that is undertaken by a [special interest group] as part of owning
+a feature or api change.  This applies to changes made to the [kubernetes/kubernetes] repo.
+
+## Design
+
+Depending on the size and impact of a feature, the design may need to go through a review process.  Changes
+that do not directly impact the api should follow the process defined by the owning sig.  Changes that
+do impact the api must go through an [api review] process.  Even non-api changes must think carefully about
+backwards compatibility to ensure clusters will continue to function correctly after an upgrade or downgrade.
+
+## Implementation
+
+Implementation includes writing and reviewing the code for the new feature.
+
+## Release tracking
+
+The owning special interest group must create an issue in [kubernetes/features] and respond to comments
+on the feature.  The feature issue MUST be updated with the current status within 1-2 days after feature freeze.
```

Comment:
Link to release process doc (https://github.com/kubernetes/community/blob/master/contributors/devel/release/README.md ?), which should describe feature freeze, etc.